backend/services/scene_memory.py

```
@@ -594,12 +585,11 @@ async def _load_shared_memory(self, user_progress_id: int, scene_id: int) -> Opt
                         progress_data = json.loads(memory.memory_content)
                         shared_context["scene_progress"].update(progress_data)
                     except (json.JSONDecodeError, ValueError) as e:
```

Ruff (0.14.1)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)